R/derive_param_maxsev_draft.R

```diff
@@ -0,0 +1,124 @@
+#' Title: DERIVE_PARAM_MAXSEV 

```

Comment:
Lets remove the title from here

Reply:
Hey Arjun,  Please look over the template for making proper roxygen documentation in the top of the file - https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#function-header-documentation